packages/AutoTDD-Tests.package/ATDDPluggableScrollableTextSpecTest.class/instance/testConfiguration.st

```diff
@@ -0,0 +1,11 @@
+as yet unclassified
+testConfiguration
+
+	| scrollableTextSpec |
+	
+	"NEWER VERSION: scrollableTextSpec := ATDDPluggableScrollableTextSpec new."
+	scrollableTextSpec := ATDDPluggableTextSpecPlus new.
```

Comment:
The newer version is now merged. Change this if possible

Reply:
Heeeellyeah